packages/eslint-plugin/src/rules/switch-exhaustiveness-check.ts

```diff
@@ -14,9 +14,11 @@ import {
   requiresQuoting,
 } from '../util';
 
+const DEFAULT_COMMENT_PATTERN = /^no default$/iu;
```

Comment:
[Feature] This should be configurable with an option, the way https://eslint.org/docs/latest/rules/default-case#commentpattern is.